tag-security-privacy.md

```diff
@@ -0,0 +1,65 @@
+# TAG Security & Privacy Review Document
+
+Answering questions from [the questionaire doc here](https://w3ctag.github.io/security-questionnaire/).
+
+## Questions to Consider
+
+### 3.1. Does this specification deal with personally-identifiable information?
+Not directly, only if Pan/Tilt is supported by the camera and the value of Pan/Tilt.
```

Comment:
Wait, this new file affects the whole spec, so this should say something else than just about `pan` and `tilt`, right?

Reply:
That's true. We can update this by -> 

_Using the values of the properties exposed, attacker might narrow down the subset of cameras probably used by the user._ 

The point is if that is _valuable enough_ information and does that need "rounding" or coarse steps/ quantization to mitigate this. With Pan/Tilt, if camera allows steps of 1 arc-degrees, we might allow implementation to allow steps of say 10 arc-degrees for all cameras, although ranges may give some hint. IMHO, since getUserMedia() is gated by user activated permission, we should try to give whatever native application can give.